src/index.js

```diff
@@ -77,6 +77,23 @@ const main = async () => {
     }
     res.status(201).send({ postId: id })
   })
+  //vote posts
+  app.put('/votes/:id', async (req,res)=>{
+    let id
+    try {
+      id = ObjectID.createFromHexString(req.params.id)
+    } catch (err) {
+      res.status(400).end()
+      return
+    }
+    const direction = req.body.direction
+    if (direction){
+      await db.collection('posts').updateOne({ _id: id }, { $inc: { upvote: 1 } })
+    } else {
+      await db.collection('posts').updateOne({ _id: id }, { $inc: { downvote: 1 } })
+    }
+    res.status(200).send({ postId: id.toHexString() })
```

Comment:
The client already knows the post ID. Maybe return the whole post object. Also return 201 instead of 200.